src/colorizer/Dataset.ts

```diff
@@ -115,6 +116,7 @@ export default class Dataset {
     this.cleanupArrayLoaderOnDispose = !arrayLoader;
     this.arrayLoader = arrayLoader || new UrlArrayLoader();
     this.features = new Map();
+    this.cachedTracks = new Map();
```

Comment:
I'm guessing this is not explicitly cleared because you just drop the whole Dataset?